src/muiTheme.tsx

```diff
@@ -56,6 +56,37 @@ const theme: any = createMuiTheme({
         color: 'black',
       },
     },
+    MuiAccordion: {
+      root: {
+        '&$expanded': {
```

Comment:
Why are we selecting `expanded` like this? You should be able to style `expanded` from outside `root` in `MuiAccordion`.
like:
```ts
MuiAccordion: {
    root: {},
    expanded: {}

}
```

Reply:
I tried and it didn't work as expected. I'm not sure, maybe because the generated expanded class is `Mui-expanded`, not `MuiAccordion-expanded`. i got the solution [here](https://github.com/mui-org/material-ui/issues/13576)

Reply:
@hav142 all good. In the future if something hacky is used from somewhere else, make sure to link it in a comment!